datasets/EU27_european_union_27_countries/EU27_european_union_27_countries.derived.ad

```diff
@@ -171,28 +171,28 @@
 - number_of_busses = 765638.0
 - number_of_trucks = 4782361.0
 - number_of_vans = 30218242.0
-- present_number_of_residences = 301431395.061
-- present_number_of_buildings = 56142611.19
-- present_number_of_apartments_before_1945 = 7500779.419
-- present_number_of_apartments_1945_1964 = 5400000.0
-- present_number_of_apartments_1965_1984 = 17900000.0
-- present_number_of_apartments_1985_2004 = 11321983.78
-- present_number_of_apartments_2005_present = 6138880.042
-- present_number_of_detached_houses_before_1945 = 35591146.0
-- present_number_of_detached_houses_1945_1964 = 17500000.0
-- present_number_of_detached_houses_1965_1984 = 50800000.0
-- present_number_of_detached_houses_1985_2004 = 37340274.96
-- present_number_of_detached_houses_2005_present = 22466009.42
-- present_number_of_semi_detached_houses_before_1945 = 0.0
-- present_number_of_semi_detached_houses_1945_1964 = 0.0
-- present_number_of_semi_detached_houses_1965_1984 = 0.0
-- present_number_of_semi_detached_houses_1985_2004 = 0.0
-- present_number_of_semi_detached_houses_2005_present = 0.0
-- present_number_of_terraced_houses_before_1945 = 12958494.44
-- present_number_of_terraced_houses_1945_1964 = 7540000.0
-- present_number_of_terraced_houses_1965_1984 = 26200000.0
-- present_number_of_terraced_houses_1985_2004 = 24683735.1
-- present_number_of_terraced_houses_2005_present = 18090091.9
+- present_number_of_residences = 208874408.0
+- present_number_of_buildings = 10111171.0
```

Comment:
This value should not change. 
When I run the etlocal migration, I see that the same value change happens. While the value has been removed from the migration.
I will try to find out why this keeps happening.

Reply:
It's fixed now